lib/crawler/cli/helpers.rb

```diff
@@ -6,6 +6,8 @@
 
 # frozen_string_literal: true
 
+require 'yaml'
+require 'erb'
 module Crawler
```

Comment:
There needs to be a new line between these, but I think this and other lint issues would be picked up if you run `make autocorrect`.

Reply:
Let me know when you've done so and I'll begin the buildkite process.

Reply:
make autocorrect passes without errors in any combination of 
```
require 'yaml'
require 'erb'
```
or
```
require 'yaml'

require 'erb'
```

Reply:
Otherwise, ive updated the PR with the requested changes, if you do want me to add a space here, regardless of the linter, let me know

Reply:
Done